README.md

```diff
@@ -550,8 +548,18 @@ See the `credits` directory for full list of contributors, credits and licenses.
 
 ## Contributing
 
-The code is available under BSD-3-Clause license
-[on github](https://github.com/composewell/streamly). Join the [gitter
-chat](https://gitter.im/composewell/streamly) channel for discussions.  Please
-ask any questions on the gitter channel or [contact the maintainer
-directly](mailto:streamly@composewell.com). All contributions are welcome!
+* [Streamly Contributor's Guide](CONTRIBUTING.md)
+* [Contact the streamly development team](mailto:streamly@composewell.com)
+
+## License
+
+Streamly is an [open source](https://github.com/composewell/streamly)
+project available under [BSD-3-Clause
+license](https://github.com/composewell/streamly/blob/master/LICENSE).
+
+## Support
+
+* Join streamly [online community chat](https://gitter.im/composewell/streamly)
+* For professional support please contact [support@composewell.com](mailto:support@composewell.com)
+
+# [Streamly Homepage](https://streamly.composewell.com)
```

Comment:
This can be removed.

Reply:
Need to have the homepage link at some prominent place. I add a reference in the intro para then.